mkdocs.yml

```diff
@@ -183,8 +183,18 @@ nav:
         - 'Web Services': 'admin/vm-ops/web-services.md'
         - 'FAQ': 'admin/vm-ops/faq.md'
     - 'Configuration Guide':
-        - 'Config TUI': 'admin/config-guide/tui.md'
-        - 'Configuration CLI':
+        - 'Config API':
+            - 'admin/config-api/README.md'
+            - 'Settings': 'admin/config-api/config.md'
+            - 'Attributes': 'admin/config-api/attribute.md'
+            - 'OpenID Client': 'admin/config-api/openid-client.md'
+            - 'Security': 'admin/config-api/security.md'
+            - 'Logs': 'admin/config-api/logs.md'
+            - 'Monitoring': 'admin/config-api/monitoring.md'
+            - 'Plugins': 'admin/config-api/plugins.md'
+        - 'CURL Guide': 'admin/config-guide/curl.md'
```

Comment:
`CURL guide` Title seems misleading. This is not going to be a guide about CURL. It is going to be a guide on how to use command line for configuring Janssen. With `configuration` being the context, `Command-line Interface` could be a better title. This will also align with `Text User Interface` title of the section below.

Reply:
No, this one is using CURL to configure the server. It already exists.

Reply:
https://docs.jans.io/head/admin/config-guide/curl/